src/sentry/api/endpoints/organization_details.py

```diff
@@ -481,4 +464,28 @@ def delete(self, request, organization):
             org_serializers.DetailedOrganizationSerializerWithProjectsAndTeams(),
             access=request.access,
         )
+
+        return context
+
+    @sudo_required
+    def delete(self, request, organization):
+        """
+        Delete an Organization
+        ``````````````````````
+
+        Schedules an organization for deletion.  This API endpoint cannot
+        be invoked without a user context for security reasons.  This means
+        that at present an organization can only be deleted from the
+        Sentry UI.
+
+        Deletion happens asynchronously and therefore is not immediate.
+        However once deletion has begun the state of a project changes and
```

Comment:
Old problem, but why does this mention projects?

Reply:
I'm guessing whoever wrote this copy/pasta'd the docstring from here: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_details.py#L644